front/app/components/FormBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -138,24 +117,22 @@ export const ContentSettings = ({
             buttonStyle="secondary"
             onClick={onClose}
             minWidth="160px"
-            width={isDeleteEnabled ? 'auto' : '100%'}
+            width={isDeleteEnabled ? '100%' : '100%'}
```

Comment:
We can remove the condition since it is 100 for both

Reply:
Oops :laughing: Thanks!!